src/versions.rs

```diff
@@ -42,6 +42,84 @@ pub struct Crate {
     pub publish: Option<bool>,
 }
 
+pub async fn get_orml_crates_and_version(
```

Comment:
Can you add a doc comment on the arguments and return types?

Reply:
On it!